install.sh

```diff
@@ -470,16 +470,29 @@ setup_selinux() {
         rpm_target=sle
         rpm_site_infix=microos
         package_installer=zypper
+        if [ "${ID_LIKE:-}" == suse ] && [ "${VARIANT_ID:-}" == sle-micro ]; then
```

Comment:
nit: do we need to check for `suse` again when we're already in a suse-specific block?